vendor/assets/javascripts/sortable.js

```diff
@@ -0,0 +1,1249 @@
+/**!
```

Comment:
i tend to reference raw js for development incase we need to debug something. it also seems more natural for the asset pipeline to handle concatenating and minifying raw files rather than files already minified